<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pomodoro</class>
 <widget class="QMainWindow" name="Pomodoro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Pomodoro</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-color: #27272f
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QStackedWidget" name="stackedWidget">
          <property name="styleSheet">
           <string notr="true">QFrame {
	background-color: #31313c;
	border-radius: 3px;
}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="page_1">
           <widget class="QFrame" name="frame_2">
            <property name="geometry">
             <rect>
              <x>101</x>
              <y>37</y>
              <width>171</width>
              <height>110</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">QFrame {
	background-color: #3d3d47;
}

QLineEdit {
	background-color:#3d3d47;
	color:	#d0d0d0;
}

QLineEdit::hover {
	background-color: #484852;
}

QPushButton {
	background-color:#3d3d47;
	border: 1px solid #54545E;
	color:	#d0d0d0;
}

QPushButton::hover {
	background-color:	#484852;
}

QLabel {
	color:	#d0d0d0;
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QLineEdit" name="time">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>-4</y>
               <width>111</width>
               <height>91</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <pointsize>43</pointsize>
              </font>
             </property>
             <property name="text">
              <string>10</string>
             </property>
             <property name="maxLength">
              <number>3</number>
             </property>
             <property name="frame">
              <bool>false</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="placeholderText">
              <string/>
             </property>
            </widget>
            <widget class="QPushButton" name="add_btn">
             <property name="geometry">
              <rect>
               <x>112</x>
               <y>1</y>
               <width>59</width>
               <height>56</height>
              </rect>
             </property>
             <property name="text">
              <string>Add</string>
             </property>
            </widget>
            <widget class="QPushButton" name="sub_btn">
             <property name="geometry">
              <rect>
               <x>112</x>
               <y>56</y>
               <width>59</width>
               <height>53</height>
              </rect>
             </property>
             <property name="text">
              <string>Sub</string>
             </property>
            </widget>
            <widget class="QLabel" name="label">
             <property name="geometry">
              <rect>
               <x>49</x>
               <y>90</y>
               <width>21</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string>min</string>
             </property>
            </widget>
           </widget>
           <widget class="QPushButton" name="start_pomodoro">
            <property name="geometry">
             <rect>
              <x>60</x>
              <y>230</y>
              <width>241</width>
              <height>24</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	border: solid;
	border-radius: 3px;
	background-color: 	#f38064;
	color: black;
}

QPushButton::hover {
background-color: #df785f;
}</string>
            </property>
            <property name="text">
             <string>Iniciar a sessão Pomodoro</string>
            </property>
            <property name="icon">
             <iconset resource="icons.qrc">
              <normaloff>:/icons/play.png</normaloff>:/icons/play.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
           <widget class="QLabel" name="label_interval">
            <property name="geometry">
             <rect>
              <x>100</x>
              <y>168</y>
              <width>161</width>
              <height>20</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">color:	#d0d0d0;</string>
            </property>
            <property name="text">
             <string>Você não terá intervalos</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
           <widget class="QLabel" name="label_hours_left">
            <property name="geometry">
             <rect>
              <x>100</x>
              <y>150</y>
              <width>171</width>
              <height>20</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #d0d0d0;</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
           <widget class="QFrame" name="frame_4">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>381</width>
              <height>291</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>381</width>
              <height>291</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QCheckBox" name="skip_pauses">
             <property name="geometry">
              <rect>
               <x>130</x>
               <y>190</y>
               <width>101</width>
               <height>18</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">color:	#d0d0d0;</string>
             </property>
             <property name="text">
              <string>Pular pausas</string>
             </property>
            </widget>
           </widget>
           <zorder>frame_4</zorder>
           <zorder>frame_2</zorder>
           <zorder>label_hours_left</zorder>
           <zorder>label_interval</zorder>
           <zorder>start_pomodoro</zorder>
          </widget>
          <widget class="QWidget" name="page_2">
           <widget class="QFrame" name="frame">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>380</width>
              <height>290</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel {
	color: #d0d0d0;
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QProgressBar" name="time_progressbar">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>110</y>
               <width>321</width>
               <height>23</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">QProgressBar {
	border: 1px solid gray;
	border-radius: 5px;
	background-color: #5a5a63;
}

QProgressBar::chunk {
	background-color: #ce725e;
}</string>
             </property>
             <property name="value">
              <number>0</number>
             </property>
             <property name="format">
              <string/>
             </property>
            </widget>
            <widget class="QPushButton" name="pause_btn">
             <property name="geometry">
              <rect>
               <x>140</x>
               <y>160</y>
               <width>31</width>
               <height>31</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
	border-radius: 15px;
	color:	#d0d0d0;
	background-color: #f38064;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="icons.qrc">
               <normaloff>:/icons/pause.png</normaloff>:/icons/pause.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </widget>
            <widget class="QPushButton" name="back_btn">
             <property name="geometry">
              <rect>
               <x>200</x>
               <y>160</y>
               <width>31</width>
               <height>31</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
	border-radius: 15px;
	color:	#d0d0d0;
	background-color: #3d3d47;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="icons.qrc">
               <normaloff>:/icons/back.png</normaloff>:/icons/back.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </widget>
            <widget class="QLabel" name="time_label">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>82</y>
               <width>321</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
            <widget class="QLabel" name="session_label">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>20</y>
               <width>221</width>
               <height>18</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel {
	color: #d0d0d0;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </widget>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame_3">
          <property name="minimumSize">
           <size>
            <width>381</width>
            <height>291</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QFrame {
	background-color: #31313c;
	border-radius: 3px;
}

QLabel {
	color: #d0d0d0;
}

QLineEdit {
	color: #d0d0d0;
}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>91</width>
             <height>16</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Meta diária</string>
           </property>
          </widget>
          <widget class="QProgressBar" name="progressBar">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>190</y>
             <width>311</width>
             <height>23</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">QProgressBar {
	border: 1px solid gray;
	border-radius: 5px;
	background-color: #5a5a63;
	color: #d0d0d0;
}

QProgressBar::chunk {
	background-color: #ce725e;
}</string>
           </property>
           <property name="maximum">
            <number>0</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
           <property name="format">
            <string>%v minuto(s)</string>
           </property>
          </widget>
          <widget class="QFrame" name="frame_5">
           <property name="geometry">
            <rect>
             <x>140</x>
             <y>40</y>
             <width>110</width>
             <height>110</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">QFrame {
	background-color: #5a5a63;
	border-radius: 25%;
}

QLineEdit {
	background-color: #5a5a63;
}</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="maximumSize">
                 <size>
                  <width>89</width>
                  <height>23</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>12</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>Meta diária</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="meta_diaria">
                <property name="maximumSize">
                 <size>
                  <width>89</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>22</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">QLineEdit {
	border: solid;
	border-radius: 10px;
}
QLineEdit::hover {
background-color:	#4F4F57;
}</string>
                </property>
                <property name="text">
                 <string>0</string>
                </property>
                <property name="maxLength">
                 <number>1</number>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_7">
                <property name="maximumSize">
                 <size>
                  <width>89</width>
                  <height>14</height>
                 </size>
                </property>
                <property name="text">
                 <string>horas</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QLineEdit" name="lineEdit">
           <property name="geometry">
            <rect>
             <x>70</x>
             <y>230</y>
             <width>113</width>
             <height>26</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="lista_sites">
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QListWidget {
	background-color: #31313c;
	border-radius: 3px;
	color: #d0d0d0;
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
